### ISSUE TITLE
basic authentication

### ISSUE THREAD

**Issue by duelistjp** [evan-buss/openbooks#136]
i for some apps open them up outside my network using cloudflared but those allow me to require usernames and passwords.  i can't seem to find a way to require this here.  do i need to set this one up in swag like plex instead?

**Comment by evan-buss** [comment]
You are correct, openbooks doesn't have any form of user accounts or passwords. You can put it behind a reverse proxy with any type of authentication you want, however. If you are using cloudflared you can take a look at their [Access](https://developers.cloudflare.com/cloudflare-one/applications/configure-apps/self-hosted-apps/) functionality. I use it for a couple of my other self-hosted services.

**Comment by chrisf4lc0n** [comment]
You can also set up an Authentik or Authelia and use that for authentication.
